### PR TITLE
Added ASPNETCORE_FORWARDEDHEADERS_ENABLED environment variable to .NET Core 2.* images

### PR DESCRIPTION
**Pull request by kichalla**
cc @anurse @tratcher

### CODE REVIEW

File: 2.0.0/Dockerfile [hunk 1]
```diff
@@ -26,6 +26,7 @@ ENV WEBSITE_ROLE_INSTANCE_ID localRoleInstance
 ENV WEBSITE_INSTANCE_ID localInstance
 ENV PATH ${PATH}:/home/site/wwwroot
 ENV ASPNETCORE_URLS=
+ENV ASPNETCORE_FORWARDEDHEADERS_ENABLED=true
```
Comment (anurse) [inline 2.0.0/Dockerfile]: Have we used `true` or `1` for other variables like this, @Tratcher?

Comment (Tratcher) [reply 2.0.0/Dockerfile]: We don't usually set env vars, but when we read them we usually accept `true/false` or `1/0`.

Comment (anurse) [reply 2.0.0/Dockerfile]: Cool, `true` sounds good then.

**Comment by Tratcher** [comment]
@bradygaster how far back do you want to take this? We should probably leave 2.0.x alone as those are out of support.

**Comment by kichalla** [comment]
@Tratcher : @bradygaster and I discussed offline and agreed to keep the flag in 2.0.* versions also as it does not have any impact to end users.

**Comment by kichalla** [comment]
@JennyLawrance You can go ahead and merge this PR accordingly

**Comment by patricklee2** [comment]
closed in favour of https://github.com/Azure-App-Service/dotnetcore-template/pull/1

**Pull request closed without merging**